examples/eval_model.py

```diff
@@ -30,6 +31,10 @@ def setup_args():
 def eval_model(parser, printargs=True):
     random.seed(42)
     opt = parser.parse_args(print_args=False)
+    # check to make sure the model file exists
```

Comment:
if model file is none that could be ok, because we have some models that don't have a file at all (IR, repeatQuery )

Reply:
let me know what you think about the new version. too messy?

Reply:
(check for `load` function, raise runtime error if model has it and no model file found)